apis/Google.Devtools.AspNet/Google.Devtools.AspNet/CloudTrace/GrpcTraceConsumer.cs

```diff
@@ -23,12 +24,18 @@ namespace Google.Devtools.AspNet
     /// </summary>
     internal sealed class GrpcTraceConsumer : ITraceConsumer
     {
-        private readonly Task<TraceServiceClient> _client;
+        // The max number of tasks that will be allowed when waiting for the client task.
+        internal const int MaxWaitingTasks = 100;
```

Comment:
Presumably this will need the same changes as error reporting? Let's concentrate on getting one of them right first, then apply the same logic. We may even want to abstract out the code, if it gets tricky.